### PR TITLE
NO-ISSUE: Add shellcheck and bashate linters

### PR DESCRIPTION
**Pull request by donpenney**

### CODE REVIEW

File: hack/shellcheck.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+VERSION="0.7.2"
```
Comment (irinamihai) [inline hack/shellcheck.sh:3]: Should we hardcode this here or have it as a variable in the Makefile and pass it to the script?

Comment (donpenney) [reply hack/shellcheck.sh:3]: It just needs to be consistent, locking the version. Having it here means a user can just run the script directly, as well. But also, this is just a clone of the same script we've got in our other repos.

**Pull request merged**